content/en/docs/setup/production-environment/tools/kubeadm/create-cluster-kubeadm.md

```diff
@@ -11,9 +11,11 @@ weight: 30
 <img src="https://raw.githubusercontent.com/kubernetes/kubeadm/master/logos/stacked/color/kubeadm-stacked-color.png" align="right" width="150px">
```

Comment:
We could change this to a Markdown image _or_ change the Hugo page layout to add the image automatically.

Also, we should copy the image into this [Git repository](https://github.com/kubernetes/website), `k/website`.

Reply:
I don't think Markdown images can be set to align to right.

A simpler way to fix this is to add a `</img>` at the end of L11.

Reply:
And if we're to copy the image into this repo, where should I put it & how to reference it here?

Reply:
> I don't think Markdown images can be set to align to right.

You'd have to use CSS. For example see “Use CSS And Special URL Parameters” in https://www.xaprb.com/blog/how-to-style-images-with-markdown/